### PR TITLE
feat(ComboBox): add ariaLabel and ariaDescribedBy props for accessibility

### PR DESCRIPTION
**Pull request by AMANDUA**
add ariaLabel and ariaDescribedBy props to ComboBox Input feild for accessibility

## Related Issue

https://github.com/momentum-design/momentum-ui/issues/198

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by AMANDUA** [comment]
Yes @bfbiggs , its possible to pass in input props but in this way

image

i can't pass ```aria-label``` without quotes.

**Comment by bfbiggs** [comment]
If the concern is the quotes and accessibility then I suggest we add those props to the Input itself. Adding to the wrapper components would then require changes to any similar wrapper components eventually. @AMANDUA @pauljeter

**Comment by AMANDUA** [comment]
Agree @bfbiggs , so i will be updating the PR

File: react/src/lib/Input/index.js [hunk 1]
```diff
@@ -260,6 +264,10 @@ class Input extends React.Component {
 }
 
 Input.propTypes = {
+  /** @prop Text to display for blindness accessibility features | '' */
```
Comment (bfbiggs) [inline react/src/lib/Input/index.js]: If your default prop is null then it should be the word null on the other side of the pipe.
```
@prop Text to display for blindness accessibility features | null
```

File: react/src/lib/Input/index.js [hunk 2]
```diff
@@ -319,6 +327,8 @@ Input.propTypes = {
 };
 
 Input.defaultProps = {
+  ariaLabel: null,
```
Comment (bfbiggs) [inline react/src/lib/Input/index.js]: Also let's keep this alphabetic.

File: react/src/lib/Input/index.js [hunk 3]
```diff
@@ -121,6 +121,8 @@ class Input extends React.Component {
 
   render() {
     const {
+      ariaLabel,
```
Comment (bfbiggs) [inline react/src/lib/Input/index.js]: Please alphabetize.

**Comment by AMANDUA** [comment]
Hi @bfbiggs I've updated the code taking care of the comments . Please review

**Pull request merged**